packages/grid-pro/src/vaadin-grid-pro-inline-editing-mixin.js

```diff
@@ -306,6 +306,10 @@ export const InlineEditingMixin = (superClass) =>
 
     /** @private */
     _onEditorFocusOut() {
+      // Ignore focusout from internal tab event
+      if (this.__cancelCellSwitch) {
+        return;
+      }
```

Comment:
Previously the flag was only checked on `keydown` in `_switchEditCell` method. However, now when we wait for the native `focusout` it is necessary to also check it here, otherwise the debouncer will stop editing.